query.css

```diff
@@ -0,0 +1,356 @@
+    padding: 10px 20px;
+    background: #fff;
+    border-radius: 10px;
+    border: 1px solid #396df2;
+  }
+
+  .desktop-default_button:hover,
+  .default_button:hover {
+    color: #fff;
+    background: #396df2;
+  }
+
+  #order-4 {
+    order: 1;
+    flex-direction: row-reverse;
+  }
+
+  #order-2 {
+    order: 3;
+    flex-direction: row-reverse;
+  }
+
+  #order-1 {
+    order: 2;
+  }
+
+  #order-3 {
+    display: none;
+  }
+
+  /* end card */
+
+  /* about */
+  .about {
+    display: flex;
+    align-items: baseline;
+    width: 100%;
+  }
+
+  .about-left-section {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    width: 40%;
+    height: 392px;
+    margin: 50px 0 0 80px;
+  }
+
+  .about-left-section .connect_button,
+  .about-left-section .social ol {
+    margin: 0;
+  }
+
+  .about-right-section {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    margin-left: 200px;
+  }
+
+  .about_ul {
+    margin-left: 100px;
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+  }
+
+  .ul_item {
+    position: relative;
+    width: 100px;
+    bottom: -20px;
+    flex-wrap: wrap;
+    right: 120px;
+    margin: 20px 5px 20px 5px;
+  }
+
+  #icons {
+    display: none;
+  }
+
+  .contact-form {
+    background: #6070ff;
+    background-image: url(./assets/desktop/contact\ form\ background\ shapes.png);
+    width: 100%;
+    height: 800px;
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-position-x: right;
+    background-position-y: 20px;
+    border-radius: 90px 0 0 0;
+    margin-top: 103px;
+  }
+
+  .contact-title {
+    text-align: center;
+    font-weight: 700;
+    font-size: 40px;
+    color: white;
+    margin-bottom: 20px;
+    padding-top: 103px;
+  }
+
+  .p-contact {
+    width: 569px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+    padding: 24px;
+  }
+
+  .contact-info {
+    display: flex;
+    flex-direction: column;
+    padding: 20px 5px 5px 5px;
+  }
+
+  .inp-form {
+    width: 100%;
+    height: 43px;
+    margin-bottom: 20px;
+    border-radius: 3px;
+    font-size: 17px;
+    border: none;
+    color: #172b4d;
+    padding: 10px 15px;
+  }
+
+  .text-area {
+    width: 450px;
+    height: 160px;
+    font-size: 17px;
+    color: #172b4d;
+    margin-bottom: 20px;
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 3px;
+    padding: 10px 15px;
+  }
+
+  .contact-bar {
+    display: none;
+  }
+
+  .btncontact {
+    width: 140px;
+    height: 48px;
+    color: #6070ff;
+    align-self: center;
+    border: none;
+    font-size: 17px;
+    font-weight: 500;
+    background-color: white;
+  }
+
+  .btncontact:hover {
+    background-color: #6070ff;
+    border: 1px solid white;
+    color: white;
+    font-size: 17px;
+    transition: 100ms;
+    cursor: pointer;
+  }
+
+  .end-line {
+    display: flex;
+  }
+}
```

Comment:
:red_circle: Kindly make sure you remove the fixed width applied on the` .container` element as it makes your design unresponsive when resizing your screen. You can see its effect in the screenshot below :arrow_down: 
![chnage](https://user-images.githubusercontent.com/26736582/200091286-3c8fe5c4-f5b6-47a2-aaab-92e0b3d45b49.gif)